powerauth-push-client/src/main/java/io/getlime/push/client/PushServerClient.java

```diff
@@ -625,17 +626,14 @@ public Response removeAndroid(Long id) throws PushServerClientException {
      * @throws PushServerClientException In case of network, response / JSON processing, or other IO error.
      *
      */
-    private <T> T getObjectImpl(String url, Map<String, Object> params, TypeReference<? extends Response> typeReference) throws PushServerClientException {
+    private <T> T getObjectImpl(String url, MultiValueMap<String, String> params, ParameterizedTypeReference<? extends Response> typeReference) throws PushServerClientException {
         try {
-            HttpResponse<String> response = Unirest.get(serviceBaseUrl + url)
-                    .header("Accept", "application/json")
-                    .header("Content-Type", "application/json")
-                    .queryString(params)
-                    .asString();
-            return checkHttpStatus(typeReference, response);
-        } catch (UnirestException e) {
-            logger.warn(e.getMessage(), e);
-            throw new PushServerClientException(e, new Error("PUSH_SERVER_CLIENT_ERROR", "Network communication has failed."));
+            ClientResponse response = webClient.get()
+                    .uri(uriBuilder -> uriBuilder.path(url).queryParams(params).build())
```

Comment:
Just to check... Does this work even in case the "base URL" contains context?

Reply:
We can test the context easily by migrating WebFlow to the new client. I will prepare this change in a branch.

Reply:
I have confirmed that the constructed URL is correct, WebFlow uses context `/powerauth-push-client` and the new client works fine.